Add initial README (taken from nucypher/nucypher docs) (#10)
nucypher/nucypher-porter: README.rst

Porter
======

Overview
--------
NuCypher Porter can be described as the *“Infura for NuCypher”*. Porter is a web-based service that performs

> nucypher-based protocol operations on behalf of applications.

Reviewer: Just a thought to sprinkle in somewhere:

*This enables applications to behave like a "light client" by delegating network intensive operations like peering and node discovery to porter*

Author: 👍

Its goal is to simplify and abstract the complexities surrounding the nucypher protocol to negate the need for

> applications to interact with it via a python client. Porter introduces the nucypher protocol to cross-platform

Reviewer: ```suggestion
applications to interact with the network directly. Porter introduces the nucypher protocol to cross-platform
```

Reviewer: > Porter introduces the nucypher protocol to cross-platform functionality including web and mobile applications.

With the evolution of nucypher-ts this is not the motivation anymore IMO.

Author: 👍
